bin/build

```diff
@@ -23,5 +23,11 @@ mkdir -p "${profile_dir}/profile.d"
 cp .profile.d/* "${profile_dir}/profile.d/"
 echo "launch = true" > ${layers_dir}/profile.toml
 
+if [ -f "{$target_dir}/export" ]; then
```

Comment:
should this be `"{$target_dir}/exports"`

Reply:
thanks!